samples/boards/siwx91x/siwx91x_sample_app/prj.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_WIFI=y
+CONFIG_NETWORKING=y
+CONFIG_WIFI_SILABS_SIWX91X_NET_STACK_OFFLOAD=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_DHCPV6=n
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_WIFI_SILABS_SIWX91X_PING=y
+CONFIG_WIFI_SILABS_SIWX91X_DNS_CLIENT=y
```

Comment:
Can you provide some numbers showing the added value of `CONFIG_WIFI_SILABS_SIWX91X_PING `and `CONFIG_WIFI_SILABS_SIWX91X_DNS_CLIENT` compared to the Zephyr native implementation.